doc/services/device_mgmt/smp_groups/smp_group_0.rst

```diff
@@ -780,3 +780,48 @@ where:
     |              | `MGMT_ERR_ENOENT` shall be returned when      |
     |              | query has no result.                          |
     +--------------+-----------------------------------------------+
+
+Bootloader Information: MCUboot
+===============================
+
+In case when MCUboot is application bootloader empty request will
+be responded with:
+
+.. code-block:: none
+
+    {
+        (str)"bootloader"      : (str)"MCUboot"
+    }
+
+Currently "MCUboot" supports querying for mode of operation:
+
+.. code-block:: none
+
+    {
+        (str)"query"           : (str)"mode"
+    }
+
+Response to "mode" is:
+
+.. code-block:: none
+
+    {
+        (str)"mode"            : (int)
```

Comment:
Should we inform that `rc` should be part of the response? Dismiss this comment if you want to define separate format for successful (just strings) and error (just rc) response.